ktor-samples/ktor-samples-static-content/pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>ktor-samples</artifactId>
+        <groupId>org.jetbrains.ktor</groupId>
+        <version>0.4.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>ktor-samples-static-content</artifactId>
+
+    <build>
+        <sourceDirectory>src</sourceDirectory>
+        <resources>
+            <resource>
+                <directory>resources</directory>
+            </resource>
+        </resources>
+    </build>
```

Comment:
It's essentially copy/paste of other sample, so I guess those need to be cleaned up too.